src/material-experimental/mdc-card/card.ts

```diff
@@ -25,13 +25,18 @@ import {
   selector: 'mat-card',
   templateUrl: 'card.html',
   styleUrls: ['card.css'],
-  host: {'class': 'mat-mdc-card mdc-card'},
+  host: {
+    'class': 'mat-mdc-card mdc-card',
+    '[class.mdc-card--outlined]': 'appearance == "outlined"'
+  },
   exportAs: 'matCard',
   encapsulation: ViewEncapsulation.None,
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class MatCard {
   // TODO(jelbourn): add `outline` option to card (supported by MDC)
+  @Input() appearance: 'outlined' | 'raised' = 'raised';
```

Comment:
nit: now that I'm thinking about it, it might be a good idea to also have a type for this so we have one place to update if we add more appearances. E.g.

```
export type MatCardAppearance = 'outlined' | 'raised'.
```